src/librustc_lint/lib.rs

```diff
@@ -336,6 +336,11 @@ pub fn register_builtins(store: &mut lint::LintStore, sess: Option<&Session>) {
             reference: "issue #52234 <https://github.com/rust-lang/rust/issues/52234>",
             edition: None,
         },
+        FutureIncompatibleInfo {
+            id: LintId::of(ILL_FORMED_ATTRIBUTE_INPUT),
+            reference: "issue #57321 <https://github.com/rust-lang/rust/issues/57321>",
```

Comment:
Remember to adjust this in favor of a dedicated issue (label as https://github.com/rust-lang/rust/labels/C-future-compatibility + T-lang + T-compiler)

Reply:
Done.